src/api/order.js

```diff
@@ -88,7 +88,9 @@ const handleSmartResponse = (data : Object, headers : {| [$Values<typeof HEADERS
 };
 
 export function getOrder(orderID : string, { facilitatorAccessToken, buyerAccessToken, partnerAttributionID, forceRestAPI = false } : OrderAPIOptions) : ZalgoPromise<OrderResponse> {
-    if (forceRestAPI && !window[LSAT_UPGRADE_FAILED]) {
+    const lsatUpgraded = LSAT_UPGRADE_FAILED in window && !window[LSAT_UPGRADE_FAILED];
```

Comment:
I think this would work as well and eliminate the need to check for the existence of the key on window.
```suggestion
    const lsatUpgraded = window[LSAT_UPGRADE_FAILED] === false;
```

Reply:
Perfect!